tests/test_monitor.py

```diff
@@ -705,148 +673,112 @@ def test_reset(self,
+                call(1, 0, '   TIMESTAMP  MESSAGE                                           ', 'green'),
                 call(2, 0, '       0.000  ExampleMessage('),
                 call(3, 0, "                  Enable: 'Enabled' -,"),
                 call(4, 0, '                  AverageRadius: 3.2 m,'),
                 call(5, 0, '                  Temperature: 250.55 degK'),
                 call(6, 0, '              )'),
-                call(29
-                     ,
-                     0, 'Filter: E                                                       ',
-                     'cyan'),
+                call(29, 0, 'Filter regex: E', 'cyan'),
+                call(29, 15, ' ', 'cyan inverted'),
+                call(29, 16, '                                                ', 'cyan'),
 
                 # '\n' pressed.
-                call(0, 0, 'Received: 3, Discarded: 1, Errors: 0'),
-                call(1,
-                     0,
-                     '   TIMESTAMP  MESSAGE                                           ',
-                     'green'),
+                call(0, 0, 'Received: 3, Discarded: 1, Errors: 0, Filter: E'),
+                call(1, 0, '   TIMESTAMP  MESSAGE                                           ', 'green'),
                 call(2, 0, '       4.000  ExampleMessage('),
                 call(3, 0, "                  Enable: 'Enabled' -,"),
                 call(4, 0, '                  AverageRadius: 3.2 m,'),
                 call(5, 0, '                  Temperature: 250.54 degK'),
                 call(6, 0, '              )'),
-                call(29
-                     ,
-                     0, 'q: Quit, f: Filter, p: Play/Pause, r: Reset                     ',
-                     'cyan'),
+                call(29, 0, 'q: Quit, f: Filter, p: Play/Pause, r: Reset                     ', 'cyan'),
 
                 # 'p' pressed. Input frame not displayed.
 
                 # 'r' pressed.
                 call(0, 0, 'Received: 0, Discarded: 0, Errors: 0'),
-                call(1,
-                     0,
-                     '   TIMESTAMP  MESSAGE                                           ',
-                     'green'),
-                call(29
-                     ,
-                     0, 'q: Quit, f: Filter, p: Play/Pause, r: Reset                     ',
-                     'cyan'),
+                call(1, 0, '   TIMESTAMP  MESSAGE                                           ', 'green'),
+                call(29, 0, 'q: Quit, f: Filter, p: Play/Pause, r: Reset                     ', 'cyan'),
 
                 # Input after reset. 'f' pressed.
                 call(0, 0, 'Received: 1, Discarded: 0, Errors: 0'),
-                call(1,
-                     0,
-                     '   TIMESTAMP  MESSAGE                                           ',
-                     'green'),
+                call(1, 0, '   TIMESTAMP  MESSAGE                                           ', 'green'),
```

Comment:
Line too long. Limit is about 85.

Reply:
ok, fixed. thanks. (be aware that this makes the tests a bit harder to grasp and to update.)